docs/reference/regions.md

```diff
@@ -9,15 +9,18 @@ When you create a cluster in Camunda 8 SaaS, you must specify a region for that
 Currently, we make these regions available for customers on the Trial, Starter, and Enterprise Plans. Enterprise customers can discuss custom regions with their Customer Success Manager.
 
 :::note
-Our Console and Web Modeler components are currently hosted in the EU. [Contact us](/reference/contact.md) if you have additional questions.
-:::
 
-Below, find a list of regions currently supported in Camunda 8 SaaS.
+- Management cluster components (Console and Web Modeler) are currently hosted in GCP (EU). [Contact us](/reference/contact.md) if you have additional questions.
+- Running on a Trial or Starter plan and want to try a different region, or interested in other regions or cloud providers? [Contact us](/reference/contact.md) as we are able to make additional regions available on request.
```

Comment:
Is it intentional to not call out the Enterprise plan here?

Reply:
This part ", or interested in other regions or cloud providers" sounds a bit redundant (asking if you need additional regions) and suggests one might be able to ask for another provider. How about we leave this part out?

Reply:
I think we should remove the "or cloud provider" part.